### PR TITLE
Add jump testing to nightly testing

### LINKED ISSUE #380: [FEA] Add jump testing to nightly for exhaustive testing
**Is your feature request related to a problem? Please describe.**
Jump has several tests which would be a good set of unite tests to run on cuOpt to catch any corner cases

This will be enabled only for nightly so it will not bother PRs.

### PR DESCRIPTION
**Pull request by rgsl888prabhu**
## Description
This PR adds jump testing to take advantage of several unit tests to bolster test suite

## Issue
closes #380 

## Checklist

- [x] I am familiar with the [Contributing Guidelines](https://github.com/NVIDIA/cuopt/blob/HEAD/CONTRIBUTING.md).
- Testing
   - [ ] New or existing tests cover these changes
   - [x] Added tests
   - [ ] Created an issue to follow-up
   - [ ] NA
- Documentation
   - [ ] The documentation is up to date with these changes
   - [ ] Added new documentation
   - [x] NA

### CODE REVIEW

**Comment by rgsl888prabhu** [comment]
@gforsyth May I get your review on this PR ?

File: ci/thirdparty-testing/run_jump_tests.sh [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    rapids-logger "Installing Julia using official installer..."
+    # Pass 'yes' to the installer to accept any prompts automatically
+    curl -fsSL https://install.julialang.org | sh -s -- --yes
+    # Add Julia to PATH for current session
+    export PATH="$HOME/.juliaup/bin:$PATH"
+    # Also add to .bashrc for future sessions
+    rapids-logger 'export PATH="$HOME/.juliaup/bin:$PATH"' >> ~/.bashrc
+else
+    rapids-logger "Julia is already installed."
+fi
```
Comment (gforsyth) [inline ci/thirdparty-testing/run_jump_tests.sh:32]: I'm not sure we should be `curl` installing Julia in CI like this.

Maybe instead a separate actions workflow that uses https://github.com/julia-actions/setup-julia

Comment (ramakrishnap-nv) [reply ci/thirdparty-testing/run_jump_tests.sh:32]: The curl command was standard way which was shown in the documentation, but I am open to switch to actions workflow, is there an example on how I can use this along with existing setup or should I add it with custom job ?

Comment (gforsyth) [reply ci/thirdparty-testing/run_jump_tests.sh:32]: Hey @rgsl888prabhu -- I don't want to make a bunch of extra work for you.  My initial reaction to `curl ... | sh` is a little wary, but that _does_ seem like the recommended install path.

Bradley pointed out that there _are_ `conda` packages for installing Julia -- can we try that quickly to see if it works?  And if it doesn't, we can stick wtih the `curl`-based install.

https://anaconda.org/conda-forge/julia

Comment (bdice) [reply ci/thirdparty-testing/run_jump_tests.sh:32]: This seems like that’s really what they recommend. They don’t accept bug reports except from these official packages.

There are no Ubuntu apt packages: https://packages.ubuntu.com/search?keywords=Julia&searchon=names&suite=plucky&section=all

There are conda packages: https://anaconda.org/conda-forge/julia

Using a reusable GitHub Action here would make it harder to reproduce this workflow locally, so I am in favor of what is implemented here.

Comment (ramakrishnap-nv) [reply ci/thirdparty-testing/run_jump_tests.sh:32]: @gforsyth Since I am building and running this in a wheel based set-up, conda based install might not be feasible.

**Review by gforsyth — APPROVED** [review]
Sounds good
